docs/community/testing.md

```diff
@@ -0,0 +1,221 @@
+---
+uri: "/docs/testing/"
+title: "Testing"
+---
+
+This document will be most useful for developers that want to contribute to WPGraphQL and want to run tests locally. 
+
+
+In order to run tests, you must [clone the plugin from Github](https://github.com/wp-graphql/wp-graphql). Downloading from Composer or Packagist will not include the dev dependencies needed to run tests.
+
+## Testing Locally with Codeception
+
+Running tests locally with Codeception is the fastest way to run tests, but it requires some initial setup on your local machine. 
+
+### Pre-requisites
+
+- Command line access
+- PHP installed and running on your machine
+- MySQL installed and running on your machine
+- Composer
+
+
+### Install WordPress Test Environment
+
+WPGraphQL includes a script to install a local test environment.
+
+
+Running this script will install WordPress to your machines `tmp` directory, and will add WPGraphQL as a plugin and activate it. 
+
+<blockquote class="wp-block-quote">
+
+<strong>NOTE: </strong>Because this installs to a tmp directory, if you restart your computer, this will go away so you would need to run this script again to setup your test environment again.
+
+</blockquote>
+
+Run the following command, replacing the variables with data for your local Database
+
+<pre class="wp-block-code lang-shell"><code>bin/install-test-env.sh $db_name $db_user $db_pass $db_host latest true</code></pre>
```

Comment:
Left this as <pre> html and it appears to render and style as expected.

Reply:
in markdown, we'd want to do this using backticks, not markup.

That way it plays nice with other markdown rendering UIs, like Github, etc

So we'd do something like

```

```shell
code goes here

```

Reply:
Then the parser would parse the markup that Github provides, take the language from the backticks and pass it to the CodeBlock component (or whatever we call it)